src/main/java/server/usecases/friendinteractors/acceptfriend/acceptFriendInteractor.java

```diff
@@ -19,15 +19,28 @@ public acceptFriendInteractor(acceptFriendDSGateway dataBase, acceptFriendOutput
     public void acceptFriend(acceptFriendInputModel model) {
         int friendid = model.getFriendid();
         int requesterid = model.getRequesterid();
-        String friendName = dataBase.getuserName(friendid);
+        String friendName = dataBase.getUserNamebyUID(friendid);
         int requesterPeerPort = dataBase.getPeerPort(requesterid);
         String requesterAddress = dataBase.getAddress(requesterid);
-        int friendPeerPort = dataBase.getPeerPort(friendid);
-        String friendAddress = dataBase.getAddress(friendid);
+//        int friendPeerPort = dataBase.getPeerPort(friendid);
+//        String friendAddress = dataBase.getAddress(friendid);
         String ifAccept = model.getIfAccept();
+        dataBase.cleanRequesterListDuplicateUID(requesterid);
+        dataBase.cleanRequesterListDuplicateUID(friendid);
         if(Objects.equals(ifAccept, "True")){
             dataBase.acceptFriendbyID(friendid, requesterid);
-            output.success(requesterid, friendid, friendName, requesterAddress, requesterPeerPort);
+
+            String[] requesterData = dataBase.readUser(requesterid);
+            String[] friendData = dataBase.readUser(friendid);
+
+            String[] toRequester = Arrays.copyOfRange(friendData, 0, 5);
+            toRequester[3] = toRequester[3].split("-")[0];
```

Comment:
Create a private chat between the friend and the requester when the friend request is accepted, send private chat to both users.